dhcpv6/dhcpv6relay_test.go

```diff
@@ -84,7 +84,7 @@ func TestDHCPv6RelayToBytes(t *testing.T) {
 	r := DHCPv6Relay{
 		messageType: RELAY_FORW,
 		hopCount:    1,
-		linkAddr:    net.IPv6loopback,
+		linkAddr:    net.IPv6zero,
```

Comment:
I'd prefer something less trivial than all-zeros and all-zeros-except-one, can you scramble up some digit? :)